Code/src/main/java/nl/utwente/group10/ui/CustomUIPane.java

```diff
@@ -110,23 +115,37 @@ private void handleScroll(ScrollEvent scrollEvent) {
 
     }
 
+    public Env getEnvInstance() {
+
+        if (envInstance == null) {
+            try {
+                // envInstance = new HaskellCatalog().asEnvironment();
+                return new HaskellCatalog().asEnvironment();
+            } catch (CatalogException e) {
+                // TODO Think of something smart to do when this happens.
+                e.printStackTrace();
+            }
+        }
+        return envInstance;
+    }
```

Comment:
We maken in `Main` al een `HaskellCatalog` aan om 'm aan het menu te voeren. Is het een idee om die catalogus door te geven aan `CustomUIPane`? Of misschien om de catalogus (en het menu) aan te maken binnen de constructor van CustomUIPane?